news/2023/2023-05-04-osu-originals-osu-taiko-world-cup-2023.md

```diff
@@ -0,0 +1,87 @@
+
+[This beatmap](https://osu.ppy.sh/beatmapsets/1980701) by [uone](https://osu.ppy.sh/users/5321719) acted as a freemod pick in the [Grand Finals map pool](/wiki/Tournaments/TWC/2023#grand-finals). If you think you can handle peak osu!taiko gameplay, give it a try.
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Aquestion - Umami Blaze
+
+Considering [**A? / Aquestion**](https://osu.ppy.sh/beatmaps/artists/173) is [an osu!taiko player](https://osu.ppy.sh/users/12011880), it's no surprise that they know exactly what it takes to create an engaging song for osu!taiko. 
+
+[Umami Blaze](https://osu.ppy.sh/beatmapsets/1976814) mapped by [youtune3](https://osu.ppy.sh/users/6442010) was part of the [Finals map pool](/wiki/Tournaments/TWC/2023#finals) as a no-mod pick. Even if speed is your specialty, this **336 BPM** map might be a bit fast for you.
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### 7_7 feat. kasane vavzed - 7D
+
+[**7_7**](https://osu.ppy.sh/beatmaps/artists/356) was [revealed as a Featured Artist](https://osu.ppy.sh/home/news/2023-04-23-new-featured-artist-7-7) alongside the [Grand Finals map pool](/wiki/Tournaments/TWC/2023#grand-finals) showcase just a couple weeks ago, pulling yet another osu!taiko resident into our Featured Artist catalogue!
+
+For pure chaos, give your best attempt to [this map](https://osu.ppy.sh/beatmapsets/1980743) created by multi-talented mapper/composer [7_7](https://osu.ppy.sh/users/7491106)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Gram - Sigrdrifa
+
+The **Grand Finals tiebreaker** has always been the climax of any tournament's map pool, so bringing [**DJ Genki / Gram**](https://osu.ppy.sh/beatmaps/artists/295) on board was a must.
+
+Prepare your hands for over 5 minutes of [this 8.5-star map](https://osu.ppy.sh/beatmapsets/1981053) created by [Nifty](https://osu.ppy.sh/users/4956097) and [rubies87](https://osu.ppy.sh/users/4949934). Try not to get any 100s on those last 4 notes too. :)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+---
+
+Of course, if you're interested in mapping any of these new tracks, they're available as pre-timed beatmap templates on each artist's listing!
+
+- [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279)
+- [**LV.4**](https://osu.ppy.sh/beatmaps/artists/214)
+- [**A? / Aquestion**](https://osu.ppy.sh/beatmaps/artists/173)
+- [**7_7**](https://osu.ppy.sh/beatmaps/artists/356)
+- [**DJ Genki / Gram**](https://osu.ppy.sh/beatmaps/artists/295)
+
+We'd also like to give [mangomizer](https://osu.ppy.sh/users/1893718) a pat on the back for coordinating the creation of these original songs with TWC's map poolers. None of this would've been possible without his help!
```

Comment:
```suggestion
We'd also like to give [mangomizer](https://osu.ppy.sh/users/1893718) a pat on the back for coordinating the creation of these original songs with TWC's mappoolers. None of this would've been possible without his help!
```